index.js

```diff
@@ -39,7 +38,7 @@ var russianPluralGroups = function (n) {
   if (lastTwo !== 11 && end === 1) {
     return 0;
   }
-  if (2 <= end && end <= 4 && !(lastTwo >= 12 && lastTwo <= 14)) {
+  if (end >= 2 && end <= 4 && !(lastTwo >= 12 && lastTwo <= 14)) {
```

Comment:
these were intentionally ordered this way; please revert these lines

Reply:
Done